tutorials/notebooks/example_pytorch_ssdlite_mobilenetv3.ipynb

```diff
@@ -0,0 +1,459 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4c261298-309f-41e8-9338-a5e205f09b05",
+   "metadata": {},
+   "source": [
+    "# Post Training Quantization a Pytorch Object Detection Model using the Model-Compression-Toolkit - A Quick-Start Guide\n",
```

Comment:
The name is too long
maybe use:
"Post Training Quantization of PyTorch Object Detection Model"